--- conf/jserv.properties.in	Fri Sep 24 09:22:50 1999
+++ conf/jserv.properties.in.rpm	Wed Sep 29 17:41:36 1999
@@ -185,7 +185,7 @@
 # IP address is allowed, everyone on that machine can execute your servlets!
 # Syntax: security.authentication=[true,false] (boolean)
 # Default: true
-security.authentication=false
+security.authentication=true
 
 # Authentication secret key.
 # The secret key is passed as a file that must be kept secure and must
@@ -193,7 +193,7 @@
 # Syntax: security.secretKey=[secret key path and filename] (String)
 # Default: NONE
 # Note: if the file could not be opened, try using absolute paths.
-#security.secretKey=@JSERV_CONF@/jserv.secret.key
+security.secretKey=@JSERV_CONF@/jserv.secret.key
 
 # Length of the randomly generated challenge string (in bytes) used to
 # authenticate connections. 5 is the lowest possible choice to force a safe
--- conf/jserv.conf.in	Wed Sep 29 08:26:58 1999
+++ conf/jserv.conf.in.rpm	Wed Sep 29 17:41:03 1999
@@ -63,8 +63,8 @@
 # Warning: if authentication is DISABLED, everyone on this machine (not just
 # this module) may connect to your servlet engine and execute servlet 
 # bypassing web server restrictions. See the documentation for more information
-#ApJServSecretKey @JSERV_CONF@/jserv.secret.key
-ApJServSecretKey DISABLED
+#ApJServSecretKey DISABLED
+ApJServSecretKey @JSERV_CONF@/jserv.secret.key
 
 # Mount point for Servlet zones
 # (see documentation for more information on servlet zones)
