--- /home/users/glen/rpm/BUILD/x86_64-linux/ApacheJServ-1.1.2/docs/Makefile.am~	2019-02-18 11:14:18.000000000 +0200
+++ /home/users/glen/rpm/BUILD/x86_64-linux/ApacheJServ-1.1.2/docs/Makefile.am	2019-02-18 11:14:22.603006830 +0200
@@ -7,10 +7,16 @@
 		protocol/*.html	\
 		future/*.html
 
-DOCFILES = $(strip $(shell ls $(EXTRA_DIST)))
-
 docdir = ${prefix}/docs
-doc_DATA= $(DOCFILES)
+doc_DATA = *.html *.txt
+doc_imagesdir = ${prefix}/docs/images
+doc_images_DATA = images/*.gif
+doc_installdir = ${prefix}/docs/install
+doc_install_DATA = install/*.html
+doc_protocoldir = ${prefix}/docs/protocol
+doc_protocol_DATA = protocol/*.html
+doc_futuredir = ${prefix}/docs/future
+doc_future_DATA = future/*.html
 
 create-dirs :
 	$(mkinstalldirs) $(DESTDIR)$(docdir) \
